cadvisor.go

```diff
@@ -35,6 +36,7 @@ import (
 	"github.com/golang/glog"
 )
 
+var argPath = flag.String("listen_path", "", "Path to listen on (UNIX socket), defaults to empty (use TCP instead)")
```

Comment:
Removing flags is a lot harder than adding them, so if we want to use a `--listen` flag, I'd prefer not to add a `--listen_path` first.

In other words, yes, would you mind adding it in this patch?